source/common/config/utility.cc

```diff
@@ -184,6 +184,22 @@ void Utility::checkApiConfigSourceSubscriptionBackingCluster(
   // Otherwise, there is no cluster name to validate.
 }
 
+Filesystem::DestinationType Utility::filesystemDestinationTypeFromProtoConfig(
+    const envoy::config::core::v3::DestinationType proto_destination_type) {
+  switch (proto_destination_type) {
+  case envoy::config::core::v3::DestinationType::FILE:
+    return Filesystem::DestinationType::File;
+  case envoy::config::core::v3::DestinationType::CONSOLE:
+    return Filesystem::DestinationType::Console;
+  case envoy::config::core::v3::DestinationType::STDERR:
+    return Filesystem::DestinationType::Stderr;
+  case envoy::config::core::v3::DestinationType::STDOUT:
+    return Filesystem::DestinationType::Stdout;
+  default:
```

Comment:
nit: remove default, move NOT_REACHED_GCOVR_EXCL_LINE outside the switch statement.

Reply:
fixed in 4b20da95d95619ed84a0c7ee698367acf8d882a2

Reply:
I ended up reverting this change because proto injects two values in the enum and then the compiler does not like the fact that there is no default in the switch statement.